drivers/ufs/ufs.c

```diff
@@ -705,11 +705,29 @@ static void ufs_enum(void)
 	}
 }
 
+static int ufs_get_device_info(struct ufs_dev_desc *card_data)
+{
+	uint8_t desc_buf[DESC_DEVICE_MAX_SIZE];
+
+	ufs_query(QUERY_READ_DESC, DESC_TYPE_DEVICE, 0, 0,
+				(uintptr_t)desc_buf, DESC_DEVICE_MAX_SIZE);
+
+	/*
+	 * getting vendor (manufacturerID) and Bank Index in big endian
+	 * format
+	 */
+	card_data->wmanufacturerid = desc_buf[DEVICE_DESC_PARAM_MANF_ID] << 8 |
+				     desc_buf[DEVICE_DESC_PARAM_MANF_ID + 1];
+
+	return 0;
```

Comment:
nit: this only ever returns 0 - could you just change to return void? As it's an internal function, if a return error value were needed in the future it could be added painlessly.

Reply:
thx for your advice. I'll fix it next patch.

Reply:
Hi, davidcunado-arm
I have create a new pull request.
https://github.com/ARM-software/arm-trusted-firmware/pull/1264